### PR TITLE
Side icon Bar added

### PR DESCRIPTION
**Pull request by vedant-jain03**
## Related Issuse

- Info about Issue or bug

Closes: #204 

#### Describe the changes you've made

I have added material ui dependency for icons and also a good side bar satisfying theme color.

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.(Specially for Backend)
- [x] My changes generate no new warnings.

## Screenshots

https://user-images.githubusercontent.com/76901313/115144730-eb9fdf00-a06b-11eb-94ec-eff1edd66419.mp4

### CODE REVIEW

**Comment by vedant-jain03** [comment]
@codewithvk Please review

File: client/src/Component/homepage/homepage.js [hunk 1]
```diff
@@ -22,7 +28,27 @@ useEffect(()=>{
...
+
+        <div className="header">
+            <img className="vector-3" src={logo} alt="vector-3" />
+            <div className="home">
+              <img className="vector-4" src={vector5} alt="vector-4" />
+              <div className="place montserrat-bold-white-24px">{place}</div>
+            </div>
+            <div className="Auth">
+            <Link to="/signin"><div className="sign-in montserrat-bold-white-24px">{signIn}</div></Link>
+            <Link to="/signup"><div className="sign-up montserrat-bold-white-24px">{signUp}</div></Link>
```
Comment (codewithvk) [inline client/src/Component/homepage/homepage.js:51]: You need to add  /div and remove error from your code.

Comment (codewithvk) [reply client/src/Component/homepage/homepage.js:51]: temp

**Review by codewithvk — CHANGES_REQUESTED** [review]
@vedant-jain03 Please do right after read in PR Template. Your code not working.

**Review by vedant-jain03 — COMMENTED** [review]
Updated

**Comment by codewithvk** [comment]
@vedant-jain03 Thanks for PR :)

**Pull request merged**